packages/daemon/src/host.js

```diff
@@ -472,13 +468,12 @@ export const makeHostMaker = ({
           )}`,
         );
       }
+
+      introduceNamesToParty(formulaIdentifier, introducedNames);
```

Comment:
Updated to match the new location `makeGuest()`. Also, `introducedNames` is never `undefined` on account of the default parameter value.